koku/masu/test/processor/ocp/test_ocp_report_summary_updater.py

```diff
@@ -239,7 +249,9 @@ def test_update_summary_tables_new_period_last_month(
 
         self.updater = OCPReportSummaryUpdater(self.schema, self.provider, self.manifest)
 
-        start_date = self.date_accessor.today_with_timezone('UTC')
+        start_date = datetime.datetime(
+            year=self.today.year, month=self.today.month, day=self.today.day
+        )
```

Comment:
Is there some reason you're dropping the TZ-aware datetime object here? (This same pattern is repeated a few times in this PR)